### DISCUSSION TITLE
想請問大家有沒有推薦的練習網站?

### DISCUSSION THREAD

**Discussion by v0ouo** [1101-GS4719-C/Python-Community-GS4719-C#1580] [Q&A] [answered]
如題，因為高中的關係，我比較常去dandanjudge和zerojudge上刷題目，最近偶爾會去Leedcode刷，想請問大家有沒有推薦的練習網站?Python題目越多越好

**Comment by lazywulf** [comment, marked as answer]
基本上啊，線上刷題目的目的通常是為了其他理由，譬如說演算法與資料結構練習。若只是要練python的話，上述的幾個online judge其實都可以喔！而且題目量很足夠。可能要請你自己去找所謂「語法題」進行練習喔！

**Reply by v0ouo** [reply]
謝謝回覆!只是我看到judge上的題目很多都是針對C++而出，所以才想請問一下而已

**Comment by eric202501** [comment]
推薦給你一個非常不錯的資源，中央資工系有開設競技程式讀書會，不僅提供多樣的題目可供練習，還有提供適合不同學習程度的教學資源，更深入還有舉辦模擬賽增進彼此之間的交流討論喔  相關資料如下https://www.facebook.com/groups/ncucp

**Reply by v0ouo** [reply]
可是資質太差實力不夠能加競程嗎?我怕我到時候一家進去就被虐爛了欸，可以請大佬帶我一起去參加競程嗎?
